### PR TITLE
Adjust download form display of region & postal code depending on country codes

### PR DESCRIPTION
**Pull request by B77Mills**
Only allow for the visibility of the regionCode & postalCode inputs when the country code is set to US, Canada or Mexico.  

Screen Shot 2023-08-02 at 11 31 16 AM
Screen Shot 2023-08-02 at 11 31 09 AM

### CODE REVIEW

**Comment by B77Mills** [comment]
[White paper form - additional fields](https://trello.com/c/HCvZdouF/330-white-paper-form-additional-fields)

File: packages/marko-web-identity-x/browser/custom-column.vue [hunk 1]
```diff
@@ -200,11 +215,25 @@ export default {
...
+    className: {
+      type: String,
+      default: '',
+    },
   },
   /**
    *
    */
   computed: {
+    countryCode() {
```
Comment (solocommand) [inline packages/marko-web-identity-x/browser/custom-column.vue:227]: You should do the same for postal code, if you are making the field show/hide based on the selected country.

File: packages/marko-web-identity-x/browser/custom-column.vue [hunk 2]
```diff
@@ -200,11 +215,25 @@ export default {
...
+      return this.user.countryCode;
+    },
+    /**
+     *
+     */
+    showContryCodeFields() {
```
Comment (solocommand) [inline packages/marko-web-identity-x/browser/custom-column.vue]: typo
```suggestion
    showCountryCodeFields() {
```

File: packages/marko-web-identity-x/browser/form/fields/address-extra.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <form-group class-name="col-md-4">
+  <form-group :class-name="className">
```
Comment (solocommand) [inline packages/marko-web-identity-x/browser/form/fields/address-extra.vue:2]: Can you verify that this prop is already present in all of the other components you added it to in `custom-column.vue`? If not, it should be added.

**Review by solocommand — CHANGES_REQUESTED** [review]

File: packages/marko-web-identity-x/browser/form/fields/address-extra.vue (hunk 3 above)
Comment (B77Mills) [reply packages/marko-web-identity-x/browser/form/fields/address-extra.vue:2]: @solocommand  this has been added to the field components that where missing it.

**Review by solocommand — APPROVED** [review]

**Pull request merged**